mros1_reasoner/CMakeLists.txt

```diff
@@ -174,6 +177,26 @@ include_directories(
 #   DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
 # )
 
+
+install(FILES requirements.txt
+  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+
+
+set(python_scripts
+  scripts/mros1_reasoner.py
+  scripts/init_models.py
+  scripts/rosmodel2owl.py
+  scripts/owl_reasoner.py
+  scripts/test_mros1_reasoner.py
+)
```

Comment:
Are these all scripts users should execute? Or should some be considered part of a Python package?